src/app/charts/chart.component.ts

```diff
@@ -62,7 +62,7 @@ export class ChartComponent implements OnChanges {
     }
 
     onExportClicked() {
-        this.csvService.downloadAsCSV(this.chartData);
+        this.dataExportService.downloadAsJSON(this.chartData);
```

Comment:
Yeah, raw response, and `chartData` isn't that. We may need to save the most recent API indicator data response in local storage or memory so that it can be pulled here. 

Note some changes to the API -> indicator data -> chartData processing pipeline in #191 due to the addition of historical data to the chart.